android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIEditText.java

```diff
@@ -96,6 +96,18 @@ private void initializeView()
 		setNestedScrollingEnabled(true);
 	}
 
+	/**
+	 * Called when set text programmatically. Add setSection to move cursor to the last position.
+	 * The method is added for TIMOB-25655 Set textfield value by coding will make cursor to the beginning of textfield
+	 * @param text New text value to be set
+	 * @param type TextView.BufferType - characteristics of the text such as static, styleable, or editable.
+	 */
+	@Override
+	public void setText(CharSequence text, BufferType type) {
```

Comment:
Jenkins now checks our code changes to see if it conforms to our code formatting rules. This causes a failure because a method's starting curly brace '{' needs to be below the method.